dist/configs/itwinjs-recommended.js

```diff
@@ -129,6 +129,20 @@ module.exports = {
         "selector": "enumMember",
         "format": null,
         "leadingUnderscore": "allow"
+      },
+      {
+        "selector": [
```

Comment:
do we have to list all these selectors here, or would `default` be good enough?

Reply:
I just tried default, but it doesn't seem to work... TBH, I copied this straight from https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/naming-convention.md#ignore-properties-that-require-quotes

Reply:
ok, I will trust their documentation then